Gemfile.lock

```diff
@@ -209,60 +209,60 @@ GEM
       sprockets-rails
       tilt
     semantic_range (2.3.0)
-    spring (2.1.0)
+    spring (2.1.1)
     spring-watcher-listen (2.0.1)
       listen (>= 2.7, < 4.0)
       spring (>= 1.2, < 3.0)
     sprockets (4.0.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
-    sprockets-rails (3.2.1)
+    sprockets-rails (3.2.2)
       actionpack (>= 4.0)
       activesupport (>= 4.0)
       sprockets (>= 3.0.0)
     thor (1.0.1)
     thread_safe (0.3.6)
     tilt (2.0.10)
-    tinymce-rails (5.4.1)
+    tinymce-rails (5.6.1)
       railties (>= 3.1.1)
-    trestle (0.9.3)
+    trestle (0.9.4)
       activemodel (>= 4.2.0)
       kaminari (>= 1.1.0)
       railties (>= 4.2.0)
       sprockets-rails (>= 2.0.0)
-    trestle-active_storage (3.0.0)
+    trestle-active_storage (3.0.1)
```

Comment:
### [trestle-active_storage](https://github.com/richardvenneman/trestle-active_storage)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.0.0 → 3.0.1

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **19** commits. See the full changes on [the compare page](https://github.com/richardvenneman/trestle-active_storage/compare/v3.0.0...v3.0.1).

These are the first 10 commits:
- (b3bec0d) [[Security] Bump kaminari from 1.2.0 to 1.2.1](https://github.com/richardvenneman/trestle-active_storage/commit/b3bec0dae244b029e3a625d7354529bc65f5e6b4)
- (c154654) [[Security] Bump websocket-extensions from 0.1.4 to 0.1.5](https://github.com/richardvenneman/trestle-active_storage/commit/c154654bc1d681fc4f5562e87074c84e93da02da)
- (87da9af) [[Security] Bump rack from 2.2.2 to 2.2.3](https://github.com/richardvenneman/trestle-active_storage/commit/87da9aff6309444ec203d31ee681751ead99badd)
- (78b4a9c) [Dont display attachment download link and delete checkbox un…](https://github.com/richardvenneman/trestle-active_storage/commit/78b4a9c7203d86c94de5844ffa914621667d540a)
- (8b878da) [Merge pull request #72 from oleg-kiviljov/attachment-persist…](https://github.com/richardvenneman/trestle-active_storage/commit/8b878dabd4bba83e2caec67a077bb220b952c4dd)
- (af94778) [Merge pull request #63 from richardvenneman/dependabot/bundl…](https://github.com/richardvenneman/trestle-active_storage/commit/af947786a216b6cb7f3d4f9892bb5e47be748a9f)
- (4920141) [Merge pull request #65 from richardvenneman/dependabot/bundl…](https://github.com/richardvenneman/trestle-active_storage/commit/4920141f291ca304a5a0dd7206059f58587cf094)
- (254d9a5) [Merge pull request #68 from richardvenneman/dependabot/bundl…](https://github.com/richardvenneman/trestle-active_storage/commit/254d9a502eddd23922d0cb73c9448d5434ca2982)
- (c357db2) [Bump rails from 6.0.3.1 to 6.0.3.4](https://github.com/richardvenneman/trestle-active_storage/commit/c357db257b103af9df5de25c23284db31a2f043f)
- (da146f4) [[Security] Bump actionpack from 6.0.3.1 to 6.0.3.4](https://github.com/richardvenneman/trestle-active_storage/commit/da146f4156e4cc5d28969fba75272dd1e9aeceea)


</details>